R/readWrite.R

```diff
@@ -0,0 +1,96 @@
+#' Outputs a GTF file, transcript-count file, and gene-count file from bambu
+#' @title transcript to gene expression
+#' @param se a summarizedExperiment object from \code{\link{bambu}}
+#' @param path the destination of the output files (gtf, transcript counts, and gene counts)
+#' @return gtf a GTF dataframe
+#' @export
+write.bambu <- function(transcript_se,path){
```

Comment:
paramters could be names (se, path)

Reply:
The parameter transcript_se has been changed to se.